Problem2

```diff
@@ -0,0 +1,62 @@
+class Solution {
+    public void gameOfLife(int[][] board) {
```

Comment:
Good Job! Please try to add related information as per the sample file provided. Good comments.